lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -155,24 +157,39 @@ class ReportRenderer {
     perfCategoryRenderer.setTemplateContext(this._templateContext);
 
     const categories = reportSection.appendChild(this._dom.createElement('div', 'lh-categories'));
+
+    ReportRenderer.smooshAuditResultsIntoCategories(report.audits, report.reportCategories);
```

Comment:
does this make more sense up in `renderReport` before passing into `_renderReport`? Then internally it'll always be the smooshed version and we won't have to remember not to use `reportCategories` above this line